src/services/user.api.ts

```diff
@@ -1,16 +1,17 @@
-import { loginInterface } from "../types/types";
-import {AxiosResponse} from "axios";
+import { IloginInterface } from "../types/types";
+import {AxiosResponse } from "axios";
 import Axios from '../services/axios.api'
 
+
 const controller = new AbortController()
 
-const login = (data:loginInterface) => {
-    Axios.post("/auth/login", data, { signal: controller.signal })
-    .then((res: AxiosResponse<any>) => {
-        console.log(res)
-    }).catch((err) => {
-        console.log(err.message)
-    })
+const login = async (payload:IloginInterface): Promise<any> => {
```

Comment:
The I on ILogin... already shows that is actually an interface. ILoginPayload or ILoginData is a better name

Also the return type in the promise should not be any. it should be the data structure of the api documentation.
if there is not any type of data yet, you must use 'unknown' type